Woof/Woof/Model/Preferences/PreferencesHandler.swift

```diff
@@ -0,0 +1,59 @@
+import Foundation
+
+/**
+ A type used for managing user preferences in the application.
```

Comment:
Please improve the documentation/the name of type so everyone can know what is the purpose of the type.

Reply:
ok, i'll fix it